CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -392,15 +393,17 @@ public protocol TestingImageDataSource: AnyObject {
         
         let cardSize = CGSize(width: prediction.image.width, height: prediction.image.height)
         if let number = prediction.number, let numberBox = prediction.numberBox {
-            let expiry = prediction.expiryObject()
-            let expiryBox = prediction.expiryBox
+            let expiry = prediction.expiryObject() ?? Expiry(string: "", month: 0, year: 0)
```

Comment:
What is this for? Is it because we can't handle optionals? This feels like a bit of a red flag to me